README.md

```diff
@@ -162,8 +162,8 @@ You can tune compression levels as you wish. Default compression levels are `1`.
 Configurable via `local.properties`:
 
 ```
-remote_build.local_gzip_level=1
```

Comment:
I don't see why, it is presented in Mac OS and most Linux distros, should work for most of users of out the box